rerun-failed-run/README.md

```diff
@@ -46,4 +48,31 @@ jobs:
           vault-addr: ${{ secrets.VAULT_ADDR }}
           vault-role-id: ${{ secrets.VAULT_ROLE_ID }}
           vault-secret-id: ${{ secrets.VAULT_SECRET_ID }}
+          notify-back-on-error: "true"
+```
+
+
+### Integration Example on the Calling Side
+
+To handle the custom error notification, you can use the following workflow configuration:
+
+```yaml
+  workflow_dispatch:
+    inputs:
+      notify_back_error_message:
+        description: \
```

Comment:
not sure what `\` means here, it will lead to this parsing, does not look like this is what you want ([playground](https://yaml-online-parser.appspot.com/)):

```
        "description": "\\ Error message if retry was not successful. This parameter is used for internal call back actions."
```

```suggestion
        description: |-
```